src/Microsoft.AspNetCore.Rewrite/Internal/IISUrlRewrite/MatchType.cs

```diff
@@ -5,8 +5,8 @@ namespace Microsoft.AspNetCore.Rewrite.Internal.IISUrlRewrite
 {
     public enum MatchType
     {
-        Pattern,
-        IsFile,
-        IsDirectory
+        pattern,
```

Comment:
This is anti C# casing rules. Why do you need to change the enum values if you're ignoring the case when parsing?